developers/weaviate/configuration/pq-compression.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Compression
+sidebar_position: 5
+image: og/docs/configuration.jpg
+# tags: ['configuration', 'compression', 'pq']
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
+import PyCode from '!!raw-loader!/_includes/code/howto/pq-compression.py';
+import PyCodeV3 from '!!raw-loader!/_includes/code/howto/pq-compression-v3.py';
+
+
+# Configure PQ Compression
+
+## Overview
+
+import PQOverview from '/_includes/pq-compression/overview-text.mdx' ;
+
+<PQOverview />
+
+import PQTradeoffs from '/_includes/pq-compression/tradeoffs.mdx' ;
+
+<PQTradeoffs />
+
+To learn how to configure PQ, follow the discussion on this page. 
+
+:::note
+Before you enable PQ, be sure to provide a set of vectors to train the algorithm. For details, see [Enable and train PQ](#step-3-load-some-training-data)
+:::
+
+## Prerequisites
+
+This Howto page uses the Jeopardy 1000 question data set. Download it here:
```

Comment:
I expected "Download it here" to be a link. If this refers to the snippet below I would consider rephrasing.